result/typetests.py

```diff
@@ -3,18 +3,15 @@
 from .result import Result, Ok, Err
 
 
-res1 = Result.Ok('hello')  # type: Result[str, int]
-if res1.is_ok():
+res1 = Ok('hello')  # type: Result[str, int]
+if isinstance(res1, Ok):
     ok = res1.ok()  # type: Optional[str]
     mapped_to_float = res1.map_or(1.0, lambda s: len(s) * 1.5)  # type: float
 else:
     err = res1.err()  # type: Optional[int]
```

Comment:
Same here,
```suggestion
err = res1  # type: Err[int]
errValue = res1.err()  # type: int
```
```

Reply:
Fixed in 2ef68c749532376dc0124233a801a9f0f2224885